products/sle/main.pm

```diff
@@ -121,6 +121,13 @@ sub cleanup_needles {
     else {
         unregister_needle_tags('ENV-OFW-1');
     }
+
+    if (get_var('PXEBOOT')) {
+        unregister_needle_tags('ENV-PXEBOOT-0');
+    }
+    else {
+        unregister_needle_tags('ENV-PXEBOOT-1');
+    }
```

Comment:
I actually though about it, but as we have same thing above written with if/else, I believe it's better to have it consistent. Even though it doesn't stop us from changing ENV-OFW scenario.